driver/pgdriver/config.go

```diff
@@ -202,3 +202,16 @@ func env(key, defValue string) string {
 	}
 	return defValue
 }
+
+// verify is a method to make sure if the config is legitimate
+// in the case it detects any errors, it returns with a non-nil error
+// it can be extended to check other parameters
+func (c *Config) verify() error {
+	if c.User == "" {
+		return errors.New(
+			`pgdriver: username is required. 
+		 	 Check your config and make sure User is not left blank.`,
```

Comment:
It is kinda strange to see sentences in error messages...

Probably this should be something like `pgdriver: User option is empty (to configure, use WithUser)`. This way we don't assume that there is a config anywhere (pgdriver.Config can't be used for configuration - it is read-only).

And we probably should add `if user == "" { panic("user is empty") }` to `WithUser`.

What do you think?

Reply:
Yes, I agree. Users should be invited to use WithUser instead and face a panic otherwise.

Reply:
:+1: Could you update the PR?

Reply:
sure!